src/parsers.rs

```diff
@@ -169,12 +181,27 @@ impl<'a> Parser<'a> {
                 // Valid u16 to be added to the address
                 head[*head_idx] = part as u16;
                 *head_idx += 1;
+
+                if *head_idx == 6 && head[5] == 0xffff && head[4] == 0 && head[3] == 0
```

Comment:
`head[0..head_idx] == [0, 0, 0, 0, 0, 0xffff]` ?

Reply:
i was not aware of that syntax. thanks